grizli/multifit.py

```diff
@@ -3709,7 +3716,7 @@ def eval_trace_shift(shifts, self, indices, poly_order, lm, verbose, fit_with_ps
 
         return chi2/self.DoF
 
-    def drizzle_grisms_and_PAs(self, size=10, fcontam=0, flambda=False, scale=1, pixfrac=0.5, kernel='square', usewcs=False, tfit=None, diff=True, grism_list=['G800L', 'G102', 'G141', 'F090W', 'F115W', 'F150W', 'F200W', 'F356W', 'F410M', 'F444W'], mask_segmentation=True, reset_model=True, make_figure=True, fig_args=dict(mask_segmentation=True, average_only=False, scale_size=1, cmap='viridis_r'), **kwargs):
+    def drizzle_grisms_and_PAs(self, size=10, fcontam=0, flambda=False, scale=1, pixfrac=0.5, kernel='square', usewcs=False, tfit=None, diff=True, grism_list=['G800L', 'G102', 'G141', 'GR150C', 'GR150R', 'F090W', 'F115W', 'F150W', 'F200W', 'F356W', 'F410M', 'F444W'], mask_segmentation=True, reset_model=True, make_figure=True, fig_args=dict(mask_segmentation=True, average_only=False, scale_size=1, cmap='viridis_r'), **kwargs):
```

Comment:
There's something inconsistent here with mixing JWST dispersers and bandpass filters.  Maybe we should call the dispersive element `dispname` throughout, which will be `FILTER` for HST, `PUPIL-FILTER` for NIRISS and `FILTER-PUPIL` for NIRCam, with a function in `utils` that generates this from a passed `Header` or `ImageHDU`.